package.json

```diff
@@ -90,9 +82,43 @@
 		"tslib": "^1.10.0"
 	},
 	"peerDependencies": {
+		"@babel/preset-env": "^7.6.3",
+		"@babel/plugin-proposal-async-generator-functions": "^7.2.0",
+		"@babel/plugin-proposal-json-strings": "^7.2.0",
+		"@babel/plugin-proposal-object-rest-spread": "^7.6.2",
+		"@babel/plugin-proposal-optional-catch-binding": "^7.2.0",
+		"@babel/plugin-proposal-unicode-property-regex": "^7.6.2",
+		"@babel/plugin-syntax-dynamic-import": "^7.2.0",
```

Comment:
it turns out that @babel/preset-env already depends on all of the proposal-* plugins that rollup-plugin-ts depends on, so we can leave all of these out.